sqlmesh/core/renderer.py

```diff
@@ -247,9 +247,11 @@ def _resolve_table(table: str | exp.Table) -> str:
             except ParsetimeAdapterCallError:
                 raise
             except Exception as ex:
-                raise ConfigError(
-                    f"Could not render or parse jinja at '{self._path}'.\n{ex}"
-                ) from ex
+                error_msg = f"Could not render or parse jinja for '{self._path}'.\n" + (
+                    extract_error_details(ex) or f"{ex}"
+                )
```

Comment:
Nit: can we just fallback to `str(ex)` within `extract_error_details` instead of needing an `or` here?